SwiftyTesseract/SwiftyTesseract/Enums/ResultIteratorLevel.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  TesseractResultIteratorLevel.swift
+//  SwiftyTesseract
+//
+//  Created by Antonio Zaitoun on 17/04/2020.
+//  Copyright © 2020 Steven Sherry. All rights reserved.
+//
+
+import Foundation
+import libtesseract
+
+/*
+ RIL_BLOCK,
+  RIL_PARA,
+  RIL_TEXTLINE,
+  RIL_WORD,
+  RIL_SYMBOL
+ */
+public enum ResultIteratorLevel: Int{
```

Comment:
Following how `EngineMode` is defined, we don't need to specify Int here, the raw value can be defined as `TessPageIteratorLevel.RawValue`.